code/data_structures/src/list/singly_linked_list/singly_linked_list.js

```diff
@@ -57,23 +57,30 @@ function LinkedList() {
   this.remove = function(element) {
     var currentNode = head;
     var previousNode;
-
+  
+    // Check if head exist
+    if(!currentNode) console.log("list does not exist.");
```

Comment:
i think it'd not be useful to just log, you need to handle it. you can simple use if-else block to not proceed next if the node isn't exist.

Reply:
Thanks for mentioning! I forgor to add return in there. Will fix it 😊